content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+
+NFS basic share creation information can be found here: [Adding NFS Shares]({{< relref "AddingNFSShares.md" >}}). Adjust access permissions using the advanced options. 
+
+## Virtualization Prerequisites
+
+In order to be able to run a Virtual Machine (VM), hardware requirements include an x86 machine running a recent Linux kernel using an Intel processor with VT extensions, or an AMD processor with SVM extensions (AMD-V). To install a VM on SCALE, you need to first research the minimum and recommended specifications for the OS you plan to use and your full use case for that VM. Allocating too many resources to a VM can cause performance on the TrueNAS SCALE system to suffer, so it is recommended to plan ahead and ensure that the SCALE system has the resources to run both itself and a full VM deployment effectively.
+
+Software requirements include an installer for the OS you intend to install on the VM, and a TrueNAS storage pool. It is recommended to have additional datasets created beneath the storage pool to further organize your VM data.
+
+Review the [Virtualization Screens]({{< relref "VirtualizationScreens.md" >}}) to determine requirements for VM installation. See [Adding Managing VMs]({{< relref "CreatingManagingVMsSCALE.md" >}}) for more information on adding or managing VMs.
+
+## App Installation Prerequisites
+
+The first time you open the **Applications** screen, the UI asks you to choose a storage pool for applications. Select a pool that has enough space for all the application containers you intend to use. Set up a new dataset before installing your applications if you want to store your application data in a location separated from other storage on your system.
+
+![AppsSettingsChoosePool](/images/SCALE/22.02/AppsSettingsChoosePool.png "Choosing a Pool for Apps")
+
+**Applications** > **Available Applications** displays official applications pre-configured for TrueNAS SCALE. These have fewer requirements for deployment than custom applications. See [Using Apps]({{< relref "UsingApps.md" >}}) for more information, as well as [Application Screens]({{< relref "AppsScreensSCALE.md" >}}).
+
+For custom applications, [Launch Docker Image Screens]({{< relref "LaunchDockerImageScreens.md" >}}) details each field in the **Launch Docker Image** wizard. Prior to beginning a custom App installation, determine the following information:
+
+* Your host networking requirements for the container including host interface
```

Comment:
Are you wanting to create a list of requirements here? What is the fist list item? host interface? Also, watch for parallel construction in this list.
This section is not clear

Reply:
Revised list of requirements to reflect parallel construction, and make content meaning clearer.